new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -0,0 +1,557 @@
+  private static final String MII_IMAGE_TAG = "v1";
+  private static final String APP_NAME = "sample-app";
+
+  // domain constants
+  private static final String DOMAIN_VERSION = "v7";
+  private static final String API_VERSION = "weblogic.oracle/" + DOMAIN_VERSION;
+
+  private static HelmParams opHelmParams = null;
+  private static V1ServiceAccount serviceAccount = null;
+  private static String opNamespace = null;
+  private static String domainNamespace = null;
+  private String domainUID = "domain1";
+  private String miiImage = null;
+
+  /**
+   * Install Operator.
+   */
+  @BeforeAll
+  public static void initAll() {
+    // get a new unique opNamespace
+    logger.info("Creating unique namespace for Operator");
+    opNamespace = createNamespace();
+    logger.info("Created a new namespace called {0}", opNamespace);
+
+    logger.info("Creating unique namespace for Domain");
+    domainNamespace = createNamespace();
+    logger.info("Created a new namespace called {0}", domainNamespace);
+
+    // Create a service account for the unique opNamespace
+    logger.info("Creating service account");
+    String serviceAccountName = createSA(opNamespace);
+    logger.info("Created service account: {0}", serviceAccountName);
+
+    // helm install parameters
+    opHelmParams = new HelmParams()
+        .releaseName(OPERATOR_RELEASE_NAME)
+        .namespace(opNamespace)
+        .chartDir(OPERATOR_CHART_DIR);
+
+    // Operator chart values to override
+    OperatorParams opParams =
+        new OperatorParams()
+            .helmParams(opHelmParams)
+            .image(OPERATOR_IMAGE)
+            .domainNamespaces(Arrays.asList(domainNamespace))
+            .serviceAccount(serviceAccountName);
+
+    // install Operator
+    logger.info("Installing Operator in namespace {0}", opNamespace);
+    assertThat(installOperator(opParams))
+        .as("Test installOperator returns true")
+        .withFailMessage("installOperator() did not return true")
+        .isTrue();
+    logger.info("Operator installed in namespace {0}", opNamespace);
+
+    // list helm releases
+    logger.info("List helm releases in namespace {0}", opNamespace);
+    assertThat(helmList(opHelmParams))
+        .as("Test helmList returns true")
+        .withFailMessage("helmList() did not return true")
+        .isTrue();
+
+    // check operator is running
+    logger.info("Check Operator pod is running in namespace {0}", opNamespace);
+    waitForCondition(operatorIsRunning(opNamespace));
+
+  }
+
+  @Test
+  @Order(1)
+  @DisplayName("Create model in image domain")
+  @Slow
+  @MustNotRunInParallel
+  public void testCreateMiiDomain() {
+    // admin/managed server name here should match with model yaml in WDT_MODEL_FILE
+    final String adminServerPodName = domainUID + "-admin-server";
+    final String managedServerPrefix = domainUID + "-managed-server";
+    final int replicaCount = 2;
+
+    // create image with model files
+    miiImage = createImage();
```

Comment:
feels like there are some assertions missing here to make sure createImage and pushImage worked as expected

Reply:
and so on through the file..

Reply:
createImage is a method in the same class which has the assertions

Reply:
same for pushImage

Reply:
yeah, this is almost ok, but not ideal -- i want to be able to read the test (and only the test) and understand what it does -- if these is shared logic, then at least make the method name more suggestive of what it does ... `createImageAndVerifySucess()` maybe?

Reply:
ok, will change the method name

Reply:
i thought some more about this overnight, and i think i am actually more strongly concerned with this pattern.  this violates one of my key tenets - i want to be able to read the test and understand what it does.  i think this is repeating a mistake from the past, hiding the assertions away in some other method.  i think we want to stop and think about this before we start doing this again. @ddsharpe @rjeberhard what do you think?